src/ONYXKEYS.js

```diff
@@ -242,6 +242,9 @@ export default {
     // This can be either "light", "dark" or "system"
     PREFERRED_THEME: 'preferredTheme',
 
+    // Denotes if the Download App Banner has been dismissed
+    SHOW_DOWNLOAD_APP_BANNER: 'showDownloadAppBanner',
```

Comment:
Sorry, didn't catch this before; the ONYXKEY for keeping the banner has disappeared in the latest commits - that's the real reason I couldn't sign in! :smile:

Reply:
thank you!! Didn't catch this in the TS migration merge